test/mann_whitney.jl

```diff
@@ -53,4 +53,22 @@ end
 @testset "Issue #39" begin
 	@test abs(pvalue(ExactMannWhitneyUTest(Float32[1:10;], Float32[2:11;])) - 0.5096) <= 1e-4
 end
+
+@testset "Issue #126 pvalues above 1" begin
+	A = [1.34937, 1.75722,0.276514, 1.04546, 1.69085, 0.738085, 2.36313 ]
+	B = [2.62325, 1.16533, 1.1327,  0.728714]
+
+	m = ExactMannWhitneyUTest(A,B)
+	p = pvalue(m; tail = :both)
+	@test p == 1
```

Comment:
I guess you could just test
```suggestion
	@test p <= 1
```
It does not see relevant that it is actually 1.

Reply:
Let's leave it. If floating point issues cause test errors we can change it to `<= 1`.